components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -13,20 +13,27 @@ import {
     WorkspaceImageBuild,
     HEADLESS_LOG_STREAM_STATUS_CODE_REGEX,
     Disposable,
+    PrebuildWithStatus,
 } from "@gitpod/gitpod-protocol";
 import { getGitpodService } from "../service/service";
+import { PrebuildStatus } from "../projects/Prebuilds";
 
 const WorkspaceLogs = React.lazy(() => import("./WorkspaceLogs"));
 
 export interface PrebuildLogsProps {
-    workspaceId?: string;
+    workspaceId: string | undefined;
+    prebuildId?: string | undefined;
+    logsEmitter?: EventEmitter;
```

Comment:
This one looks like a copy'n'paste thing; it's not used (anymore).

Reply:
Thanks for the catch! Will remove it.